guide/samples/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name                 = "user-guide-samples"
```

Comment:
nit: I don't think we will ever release this crate, but might still be good to have google-cloud in the name.

Reply:
I will leave as-is. We have precedent (`integration-tests`, and `check-copyright`) for not using the prefix on unreleased crates.  I think the shorter names are more ergonomic.